exampleCourse/questions/demo/autograder/c/factorial/tests/main.c

```diff
@@ -0,0 +1,27 @@
+#include <check.h>
+
+extern int factorial(int x);
+
+START_TEST(test_factorial)
+{
+  ck_assert_int_eq(factorial(0), 1);
+  ck_assert_int_eq(factorial(1), 1);
+  ck_assert_int_eq(factorial(2), 2);
+  ck_assert_int_eq(factorial(5), 120);
+  ck_assert_int_eq(factorial(10), 3628800);
+}
+END_TEST
+
+int main(void)
+{
```

Comment:
Nit, for consistency with the other C files. Also consider matching the indentation (4 spaces).

```suggestion
int main() {
```